src/components/AppSettingsSection/AppSettingsSection.vue

```diff
@@ -0,0 +1,62 @@
+<!--
+ - @copyright Copyright (c) 2020 Marco Ambrosini <marcoambrosini@pm.me>
+ -
+ - @author Marco Ambrosini <marcoambrosini@pm.me>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->`
+
+<template>
+	<div :id="id" class="app-settings-section">
+		<h2 :id="title"
```

Comment:
h2 is most likely too high?
Should probably be h3 and the whole settings should have one h2 stating 'Settings for xxx' or something that define the Settings Modal as its own context.

https://www.w3.org/WAI/tutorials/page-structure/headings/

Reply:
@jancborchardt thoughts?

Reply:
Yeah, basically the dialog title e.g. "Talk settings" should be h2, meaning that any subcategory should be h3.